package.json

```diff
@@ -32,7 +32,9 @@
     "docs:prod": "npm run build -- docs --prod --aot --base-href '/nebular/'",
     "docs:dirs": "gulp create-docs-dirs",
     "docs:gh-pages": "npm run docs:build && npm run ngh -- --dir ./docs/dist",
-    "build": "ng build",
+    "docs:build:stats": "npm run docs:prod -- --stats-json",
+    "docs:analyze": "webpack-bundle-analyzer docs/dist/stats.json",
+    "build": "node --max_old_space_size=8048 ./node_modules/@angular/cli/bin/ng build",
```

Comment:
Why do we need to call angular-cli this way?

Reply:
as `node` was running out of memory.